libs/bootstrap/src/lib/setup-web-app.ts

```diff
@@ -112,6 +126,33 @@ function setupGlobalInterceptors(app: INestApplication) {
   app.useGlobalInterceptors(new LoggerErrorInterceptor());
 }
 
+async function runDatabaseSeeders(
+  app: INestApplication,
+  logger: Logger,
+  shouldRunSeeds: boolean,
+) {
+  if (!shouldRunSeeds) {
+    return;
+  }
+
+  const ds = callOrUndefinedIfException(() => app.get(DataSource));
+  const seeders = app.get(TYPEORM_SEEDERS_TOKEN);
+  const factories = app.get(TYPEORM_FACTORIES_TOKEN);
+
+  if (seeders.length === 0) {
+    return logger.warn(
+      'Warning: No seeders found. Ensure you have provided seeders if you are expecting database seeding to occur.',
+    );
+  }
+
+  if (ds instanceof DataSource) {
```

Comment:
else log.warn('Seems like run seeds is enabled, but there is no data source provided, this seems like a mistake. Please review or disable seed run')